docs/contributors/versions-in-wordpress.md

```diff
@@ -6,7 +6,8 @@ If anything looks incorrect here, please bring it up in #core-editor in [WordPre
 
 | Gutenberg Versions | WordPress Version |
 | ------------------ | ----------------- |
-| 8.5.1-9.2            | 5.6             |
+| 8.6-9.2            | 5.6             |
+| 8.6-9.2            | 5.5.2             |
```

Comment:
I think this should be the same as the line below for 5.5.1:
```suggestion
| 7.6-8.5            | 5.5.2             |
```

There will be a few small bug fixes cherry-picked from beyond plugin version 8.5 in 5.5.2 (in addition to the ones from 5.5.1):
https://core.trac.wordpress.org/timeline?from=2020-10-29T01%3A05%3A28Z&precision=second

Reply:
Ah ha! You're spot on. I got confused by looking [at this line](https://github.com/WordPress/wordpress-develop/blob/5.5.2/package.json#L103) rather[ than this one](https://github.com/WordPress/wordpress-develop/blob/5.5.2/package.json#L102) :D